### PR TITLE
Add RBAC database migration documentation

### PR DESCRIPTION
**Pull request by mcarmona99**
## Description

This pull request closes https://github.com/wazuh/wazuh-documentation/issues/5058.

In this PR, I have added the RBAC database update process documentation.

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns.

### CODE REVIEW

File: source/user-manual/reference/tools/rbac-control.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+.. _rbac_control:
+
+rbac_control
+============
+
+.. versionadded:: 4.5.0
+
+The ``rbac_control`` tool allows changing the default users' password and reseting the RBAC database to its default
+state. For more information about the Wazuh RBAC resources and database, please visit the
```
Comment (vicferpoy) [inline source/user-manual/reference/tools/rbac-control.rst:11]: Same as above. I would use a more generic sentence.

File: source/user-manual/reference/tools/rbac-control.rst [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+-----
+
++-----------------------------------------+---------------------------------------------------+
+| Option name                             | Option description                                |
++=========================================+===================================================+
+| ``-h, --help``                          | Display the help message.                         |
++-----------------------------------------+---------------------------------------------------+
+| ``change-password``                     | Change the password for each default user.        |
++-----------------------------------------+---------------------------------------------------+
+| ``factory-reset``                       | Reset the RBAC database to its default state.     |
```
Comment (vicferpoy) [inline source/user-manual/reference/tools/rbac-control.rst:24]: Look for a way to add that this action requires a confirmation message unless the `-f` or `--force` flag is added.

File: source/user-manual/reference/tools/rbac-control.rst [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+.. code-block:: console
+    :class: output
+
+    New password for 'wazuh' (skip):
+    New password for 'wazuh-wui' (skip):
+        wazuh: FAILED | Error 5007 - Insecure user password provided
+
+
+``change-password`` example where the password was changed successfully:
```
Comment (vicferpoy) [inline source/user-manual/reference/tools/rbac-control.rst:91]:
```suggestion
``change-password`` example where the `wazuh` user password was changed successfully (to skip any of the user, leave the new password blank):
```

**Review by vicferpoy — CHANGES_REQUESTED** [review]

File: source/user-manual/reference/tools/index.rst [hunk 4]
```diff
@@ -29,6 +29,8 @@ Tools
 +---------------------------------------------------+----------------------------------------------------------------------------+-----------------------------+
 | :doc:`wazuh-regex <wazuh-regex>`                  | Validates a regex expression                                               | manager                     |
 +---------------------------------------------------+----------------------------------------------------------------------------+-----------------------------+
+| :doc:`rbac_control <rbac-control>`                | Manage default API users' password and reset RBAC DB                       | manager                     |
```
[the remark this answers is not in the document]
Comment (mcarmona99) [reply source/user-manual/reference/tools/index.rst:32]: Giving a more generic summary.

File: source/user-manual/reference/tools/rbac-control.rst [hunk 5]
```diff
@@ -0,0 +1,102 @@
+.. Copyright (C) 2022 Wazuh, Inc.
+
+.. _rbac_control:
+
+rbac_control
+============
+
+.. versionadded:: 4.5.0
+
+The ``rbac_control`` tool allows managing resources from the Wazuh RBAC database and reseting the DB to its default
```
Comment (vicferpoy) [inline source/user-manual/reference/tools/rbac-control.rst:10]:
```suggestion
The ``rbac_control`` tool allows managing resources from the Wazuh RBAC database and resetting the DB to its default
```

**Review by vicferpoy — CHANGES_REQUESTED** [review]

**Pull request merged**